[Publish 2-21 after PM confirmation] What's new: Live archives (#16159)
newrelic/docs-website: src/content/whats-new/2024/02/whats-new-02-21-live-archives.md

- **Reduce costs**: Live archives eliminates the need to pay for ingress and egress, indexing, reformatting data, or the need for additional tools to manage compliance risks. 

![Screenshot of the retention management UI, showing live archives](./logs-live-archives-retention.png "Screenshot of the retention management UI, showing live archives")

## How to get started

New Relic live archives simplifies the way you approach long-term logs storage and analysis. Ready to streamline your logs process? Follow these steps:  

1. **Send logs**: Send your logs to New Relic.
1. **Set retention periods**: Choose a retention period from 120 days up to seven years. This depends on your compliance needs.
1. **Choose a partition to archive**: You don't need to archive every log for compliance. Your New Relic administrator can select specific log partitions for archiving.
1. **Query data in the logs UI**: Live archives logs are queryable alongside your other logs and telemetry data.

> 2. **Manage user permissions**: To query your live archives, have a New Relic administrator assign you a custom role so you can access query permissions. 

Author: ```suggestion
2. **Manage user permissions**: To query your live archives, have a New Relic administrator assign you a custom role with query capability.
```

I'm thinking "permissions" aren't something you "access", so much as you "have" the "capability"--that's a term used in our RBAC system for "permission to do a specific thing". Looks great overall!
